Gloss of top docs pages (#28891)
hashicorp/terraform: website/intro/index.html.markdown

## Next Steps

> - Learn about common [Terraform use cases](/intro/use-cases.html) to see the
> multiple ways Terraform can be used. 

Reviewer: ```suggestion
- Learn about common [Terraform use cases](/intro/use-cases.html) to find multiple
ways you can use Terraform.
```

> - Learn [how Terraform compares to other software](/intro/vs/index.html) to
>   compare Terraform to other infrastructure tools. 

Reviewer: This is kinda redundant 
```suggestion
- Learn [how Terraform compares to other infrastructure tools](/intro/vs/index.html). 
```
